examples/server_events.rs

```diff
@@ -49,6 +49,11 @@ fn client_changed_event_handler(ctx: &Context, client_event: ClientChangeSubeven
     }
 }
 
+#[shutdown_event_handler]
+fn shutdown_event_handler(ctx: &Context, _event: u64) {
```

Comment:
If there is an integration test, we can validate server shut down using the `SHUTDOWN` command from a client.

However, the test will have to validate something was done. For example, it can search and check that the engine logs contains X message in the log file